### PR TITLE
Add more info on eve-tsc contribution

### PR DESCRIPTION
**Pull request by zedi-pramodh**
Signed-off-by: Pramodh Pallapothu <pramodh@zededa.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list at](https://lists.lfedge.org/g/eve-tsc/)
```
Comment (eriknordmark) [inline CONTRIBUTING.md:39]: We should also add the https://lists.lfedge.org/g/eve mailing list.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ composed of the following members:
...
+and also susbscribe to the [eve-tsc Calendar at](https://lists.lfedge.org/g/eve-tsc/calendar)
+
```
Comment (eriknordmark) [inline CONTRIBUTING.md:41]: Also specify the slack channels #eve and #eve-help at https://lists.lfedge.org/g/eve and provide a link to the wiki (https://wiki.lfedge.org/display/EVE/EVE)

FWIW I found those at the bottom of https://www.lfedge.org/projects/eve/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,6 +36,9 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list at](https://lists.lfedge.org/g/eve-tsc/)
+and also susbscribe to the [eve-tsc Calendar at](https://lists.lfedge.org/g/eve-tsc/calendar)
```
Comment (eriknordmark) [inline CONTRIBUTING.md:40]: There are two calendars; the other is https://lists.lfedge.org/g/eve/calendar
I see that some events appear on both but it makes sense listing both of them here.

**Comment by zedi-pramodh** [comment]
Amended the commit

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -36,6 +36,15 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list](https://lists.lfedge.org/g/eve-tsc/)
+and [eve mailing list](https://lists.lfedge.org/g/eve)
+
+You can susbscribe to the [eve-tsc Calendar](https://lists.lfedge.org/g/eve-tsc/calendar)
+and [eve Calendar](https://lists.lfedge.org/g/eve/calendar)
+
+You can subscribe to slack channels [eve and eve-help](https://lists.lfedge.org/g/eve)
```
Comment (eriknordmark) [inline CONTRIBUTING.md:45]: My bad: the slack channel link should be https://slack.lfedge.org/

**Review by eriknordmark — APPROVED** [review]
A copy-paste error from me to fix.

File: CONTRIBUTING.md (hunk 4 above)
Comment (zedi-pramodh) [reply CONTRIBUTING.md]: np, amended the commit

**Pull request merged**